Remove "new" & redundant text, add "recommended" (#2938)
nginxinc/kubernetes-ingress: docs/content/configuration/virtualserver-and-virtualserverroute-resources.md

---
title: VirtualServer and VirtualServerRoute Resources

> description: "The VirtualServer and VirtualServerRoute resources are load balancing configurations offered as a recommended alternative to the Ingress resource."

Reviewer: ```suggestion
description: "The VirtualServer and VirtualServerRoute resources are load balancing configurations recommended as an alternative to the Ingress resource."
```
